README.rst

```diff
@@ -158,6 +158,8 @@ Common parameters:
                         migrations are run against. This option must be used
                         in conjuction with the -k option. This option is
                         ignored unless the -s option is provided.
+  -v PROTOCOL_VERSION, --protocol-version PROTOCOL_VERSION
```

Comment:
I would like to avoid `-v` as an argument name, as it is commonly associated with verbosity.

Reply:
Agreed. I just removed the -v option.